app/assets/stylesheets/petitions/views/_search.scss

```diff
@@ -3,6 +3,45 @@
   margin-top: $gutter;
 }
 
+.search-inline {
+  @extend %contain-floats;
+
+  .form-control {
+    @include box-sizing(border-box);
+    @include appearance(none);
+    float: left;
+    min-width: 0;
+    margin: 0;
+    border-right: 0;
+    height: 40px;
+    padding: 6px 0;
+    width: 85%;
+    width: -webkit-calc(100% - 40px);
+    width: calc(100% - 40px);
```

Comment:
I have to wrap the `$calc` variable in quotes or it doesn't compile. Still cool?